activestorage/test/models/variant_test.rb

```diff
@@ -172,6 +172,25 @@ class ActiveStorage::VariantTest < ActiveSupport::TestCase
     assert_equal 8, image.height
   end
 
+  test "resized variation of PNG blob without format should be PNG type" do
+    blob = create_file_blob(filename: "png_image_without_format", content_type: "image/png")
+    variant = blob.variant(resize: "15x15").processed
+
+    assert_equal "PNG", read_image(variant).type
+  end
+
+  test "resized variation of JFIF works for vips processor" do
+    ActiveStorage.variant_processor = :vips
+    blob = create_file_blob(filename: "image.jfif", content_type: "image/jpeg")
+    assert_nothing_raised do
+      blob.variant(thumbnail_image: 100).processed
+    end
+  rescue LoadError
+    # libvips not installed
```

Comment:
Instead of silently passing the test, I think it'll be better to skip with a message here.

Reply:
@tgxworld This is how the current codebase handle it on other `vips` tests but I'm agree with you that skip with a message is better, I will add it to all the tests.

Reply:
@tgxworld Done.